bin/pd/Makefile

```diff
@@ -317,6 +316,7 @@ rebuild-grants: $(workdir)/filtered/grants.csv $(workdir)/filtered/grants-nil.cs
 	"$(workdir)/filtered/grants.csv" "$(workdir)/amendments/grants.csv"
 	$(paster) --plugin=ckanext-canada grants rebuild --lenient -c $(portal_ini) \
 	-f "$(workdir)/amendments/grants.csv" "$(workdir)/filtered/grants-nil.csv"
+	$(ogc_search)/load_gc_to_solr.py "$(workdir)/filtered/grants.csv" "$(workdir)/filtered/grants-nil.csv"
```

Comment:
did you mean to use `$(workdir)/amendments/grants.csv` here or have you reimplemented the amendment-diff logic?